README.md

```diff
@@ -37,10 +38,69 @@ To publish this example to your own Pactflow account and view it in the UI follo
     ```make verify_swagger```
 
 4. If the Schemathasis test is successful run the publish_contract target, this will publish the Swagger document, Schemathesis report, and success status of the verification to your Pactflow account
-
-### Using the fake CI
+   
+5.  Using the fake CI
 * `make test` - run the tests locally
-* `make fake_ci` - run the CI process locally
+* `make fake_ci` - run the CI process locally, this will publish the Swagger document and Shchemathesis report to Pactflow.
+ 
+### Using Example on Windows.
+You can still try this example locally on Windows using powershell and running commands manually. These will be the same commands that are used in the makefile with a few manual tweaks.
+
```

Comment:
To run this example on Windows, we will use Powershell and run the commands defined in the Makefile, manually. 

We will take you through the steps, but you can see the commands in the `./Makefile`

Some notes

- Most of them use the pact-cli docker image
- can be ran locally from Powershell by changing the unix paths to windows paths
- Replace the environment variable references. Any variable referenced as `${VARIABLE}` can be changed to `$env:VARIABLE` for use Powershell.